src/modules/rating/rating.ts

```diff
@@ -0,0 +1,26 @@
+import { ObjectId } from 'mongodb';
```

Comment:
Lets make the rating a separate PR.
1. On dev branch `git checkout -b rating-feature` (creates a new branch off the current one)
2. `git push origin rating-feature`
3. `git checkout dev`
referenced this for the steps after because I don't know this off the top of my head: https://stackoverflow.com/questions/65209693/how-to-revert-back-my-github-commit-to-the-previous-one
4. `git reset ef7ddd8 ` this is the hash of your commit right before you added ratings
5. `git push --force origin`

Then open a second PR with just the rating stuff; `rating-feature` -> `dev`